inspirehep/migrator/api.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import zlib
+
+from flask import current_app
+from redis import StrictRedis
+from redis_lock import Lock
+
+from inspirehep.migrator.tasks import (
+    insert_into_mirror,
+    migrate_from_mirror,
+    wait_for_all_tasks,
+    LOGGER,
+)
+
+
+def continuous_migration():
```

Comment:
I would log a lot more so we can get nice graphs in grafana and know what's going on.

Reply:
done :+1: